src/gui/uithememanager.cpp

```diff
@@ -37,14 +37,28 @@
 #include <QPalette>
 #include <QResource>
 
-#include "base/iconprovider.h"
 #include "base/logger.h"
 #include "base/preferences.h"
 #include "base/utils/fs.h"
 
 namespace
 {
+    const QString ICONS_DIR = QStringLiteral(":icons/");
+    const QString THEME_ICONS_DIR = QStringLiteral(":uitheme/icons/");
     const QString CONFIG_FILE_NAME = QStringLiteral(":uitheme/config.json");
+
+    QString findIcon(const QString &iconId, const QString &dir)
+    {
+        const QString pathSvg = dir + iconId + ".svg";
```

Comment:
`QLatin1String(".svg")`

Reply:
I guess this code is used a lot so I'm wrapping it around QStringLiteral.

Reply:
I don't think `QStringLiteral` will be more suitable than `QLatin1String` here, the entire string still has to be built/concatenated at run time.